src/config/dynamic-theme-fixes.config

```diff
@@ -19001,19 +19001,19 @@ i.ml-1
 steamdeck.com
 
 INVERT
-section#available-now div.col_4.copy
-section#available-now h1
+section#available-now div.col_9
 section#experience div.col_8
 section#experience h2
 section#topfeature div.col_5
 
+CSS
+section#available-now {
+    background-color: #f0f0f0 !important;
```

Comment:
Can this be made in the format of `${color}`, see the table at https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md#how-to-use-the-dev-tools.

Reply:
Fixed. Good catch!